src/CoreWCF.WebHttp/src/CoreWCF/OpenApi/OpenApiSchemaBuilder.cs

```diff
@@ -58,6 +58,13 @@ public static OpenApiDocument BuildOpenApiSpecificationDocument(OpenApiOptions i
             PopulateOpenApiInfo(document, info);
             PopulateOpenApiPathsOperations(document, contracts, info.TagsToHide);
 
+            if (info.TagsSorter != null)
+            {
+                var tags = document.Tags.ToList();
+                tags.Sort(info.TagsSorter);
```

Comment:
You can avoid an allocation here as Tags defaults to a `List<OpenApiTag>`. In case someone else replaces it or they change the implementation, you need the ToList as a backup, but the following code should avoid an allocation.
```c#
var tags = document.Tags as List<OpenApiTag> ?? document.Tags.ToList();
```

Reply:
I pushed up the changes for this.